.github/workflows/portage-stable-packages-list

```diff
@@ -502,6 +501,7 @@ net-dialup/lrzsz
 net-dialup/minicom
 
 net-dns/bind
+net-dns/bind-tools
```

Comment:
I would have just changed it to `bind` in the sssd ebuild. The automation scripts will undo this, but you can easily skip that change for a short time.

Reply:
Welp, I'm wondering how short this "short time" will be. I can see that `net-dns/bind-tools` ebuilds are quite usable for versions <9.18.0, i.e. they are not wrappers around net-dns/bind, so my impression is that as long as there are ebuilds for 9.16.x, net-dns/bind-tools will stay. And people using sys-auth/sssd together with net-dns/bind-tools-9.16.x (because they masked bind 9.18.x) would be upset if you just changed the dependency from bind-tools to bind. Maybe we could ask for updating the dependency in sssd to be `<net-dns/bind-tools-9.17 || >net-dns/bind-9.17`?

Reply:
My guess is they're just waiting for this [stabilisation bug](https://bugs.gentoo.org/941696) so that they can just point things directly at net-dns/bind and eventually drop all the old versions. I don't feel strongly about it so you choose.